.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+repos:
+-   repo: https://gitlab.com/pycqa/flake8
```

Comment:
Any reason you used this repo versus `https://github.com/pre-commit/pre-commit-hooks`? Curious because that's what I used on all the other pre-commit hooks (without much rationale tbh) so if there's a good reason to go with a different repo would be good to know!

Reply:
Yes, I looked at the documentation on the [pre-commit-hooks repo](https://github.com/pre-commit/pre-commit-hooks), and it pointed to the gitlab repo for flake8 in the Deprecated / replaced hooks section.

Reply:
Thanks! You obviously read that more closely than I did!